Userland/Applications/DisplaySettings/MonitorSettingsWidget.h

```diff
@@ -19,7 +19,14 @@ class MonitorSettingsWidget : public GUI::Widget {
     C_OBJECT(MonitorSettingsWidget);
 
 public:
+    virtual ~MonitorSettingsWidget() override
```

Comment:
```suggestion
    ~MonitorSettingsWidget() override
```

`virtual` is redundant when you're using `override` (because you can't override non-virtual methods).

Reply:
Just as a note, the coding style document mandates specifying both `virtual` and `override`:
https://github.com/SerenityOS/serenity/blob/master/Documentation/CodingStyle.md#overriding-virtual-methods